package/src/skia/Canvas.ts

```diff
@@ -4,21 +4,26 @@ import type { IFont } from "./Font";
 import type { IPath } from "./Path";
 import type { IImage } from "./Image";
 import type { SVG } from "./SVG";
-import type { Color } from "./Color";
+import type { IColor } from "./Color";
```

Comment:
Just a small note about the prefix - is there a specific reason for this prefix, wouldn't it make sense to just avoid the `I` all over (like we discussed with the animation/value types)